### PR TITLE
Create skeleton for 2017.28 release notes

### PR DESCRIPTION
**Pull request by richardmateosian-okta**
## Description:
Create skeleton for 2017.28 platform release notes
### Resolves:
* https://oktainc.atlassian.net/browse/OKTA-132200

### CODE REVIEW

File: _source/_docs/platform-release-notes/platform-release-notes.md [hunk 1]
```diff
@@ -17,49 +17,64 @@ Okta SDK `EventsAPIClient`.
 The new data retention policy starts:
 
 * June 7, 2017 for existing preview orgs
-* July 17, 2017 for existing production orgs
+* July 17, 2017 for existing production org
```
Comment (richardmateosian-okta) [inline _source/_docs/platform-release-notes/platform-release-notes.md]: I don't know where that final "s" went. I'll put it back.

Comment (mystiberry-okta) [inline _source/_docs/platform-release-notes/platform-release-notes.md]: FYI you can commit and push without creating a doc PR. I typically don't create the doc PR until it's ready for review.

**Comment by mystiberry-okta** [comment]
We hammered out a structure on Friday but it ended up putting too much emphasis on the details that aren't that important to customers--as far as most of them know, "GA" is the first time they'll see the feature or feel its effects unless they asked for the FF to be enabled. So, here's a suggested restructure. It's annoying to mention dates in so many places that will have to be updated every week, but I see no way around it and still keep the customer oriented in the RN, keeping them easily scannable. If there's a better way of doing it, go for it. But the key things for the customer is: what's hitting my prod org when, and what will it do?
suggested-restructure

**Comment by mystiberry-okta** [comment]
I should have added a # to those last three items before Platform Bugs Fixed.
Also note that content is missing for the OpenID Connect (I put some suggested content in the JIRA) and key rollover.

**Comment by mystiberry-okta** [comment]
Hoping this suggested new structure is helpful. It de-emphasizes the lifecycle (which customers don't really care about) and emphasizes WHEN changes are coming. Also gets the titles shorter.

new-structure

**Comment by mystiberry-okta** [comment]
@richardmateosian-okta Please incorporate Jeff's info about data retention. You may well want to add a heading for "Generally Available" and one for "Early Availability" given the number of items and the fact that Data Retention is moving out of "Advance Notice".

**Pull request merged**